packages/core/src/sass/app/_common/components/account-switcher.scss

```diff
@@ -183,6 +187,30 @@
             height: 2.8rem !important;
         }
     }
+
+    &__wallets {
+        padding: 1.2rem 1.6rem;
+
+        &-container {
+            align-items: center;
+            -webkit-box-align: center;
```

Comment:
I dont think this is neccessary. Maybe substitute it with something else like `justify-content: center`